scripts/entrypoint.sh

```diff
@@ -2,6 +2,16 @@
 
 echo "starting jupyter notebook"
 
+if [ -n "$SPARK_DRIVER_HOST" ]; then
+    echo "Setting spark.driver.host to $SPARK_DRIVER_HOST"
+    source /opt/bitnami/scripts/spark-env.sh
+    if [ -z "$SPARK_CONF_FILE" ]; then
```

Comment:
I think the previous attempt to write to the Spark config file failed because we didn't set `$SPARK_CONF_FILE`.